internal/api/v1/applications.go

```diff
@@ -157,6 +172,165 @@ func (hc ApplicationsController) Update(w http.ResponseWriter, r *http.Request)
 
 	return nil
 }
+func (hc ApplicationsController) Logs(w http.ResponseWriter, r *http.Request) APIErrors {
+	params := httprouter.ParamsFromContext(r.Context())
+	org := params.ByName("org")
```

Comment:
I think you need to check the validity of org and app before you start doing websocket things. Check how other actions return 404 when org or app don't exist.